perf/istio-install/values-istio-postsubmit.yaml

```diff
@@ -78,6 +80,11 @@ gateways:
     - port: 5204
       name: tcp-iperf-tls
 
+policy:
```

Comment:
doesn't this need to be mixer.policy?

Reply:
No, it is for  istio-policy.

Reply:
https://github.com/istio/istio/blob/a952305f6c8489a90515a11bc4ac10b83ced39d2/install/kubernetes/helm/istio/values.yaml#L35-L38
policy is always nested under mixer I think

Reply:
agreed policy should be under mixer.